third-party/qthread/Makefile

```diff
@@ -235,6 +243,12 @@ qthread-chapel-h: FORCE
 	echo "#define CHPL_QTHREAD_HAVE_GUARD_PAGES" \
 	     $(HAVE_GUARD_PAGES) \
 	     >> $(QTHREAD_INSTALL_DIR)/include/qthread-chapel.h
+	echo "#define CHPL_QTHREAD_TOPOLOGY_BINDERS" \
+       $(USE_TOPOLOGY_BINDERS) \
+       >> $(QTHREAD_INSTALL_DIR)/include/qthread-chapel.h
+	echo "#define CHPL_QTHREAD_TOPOLOGY_HWLOC" \
+       $(USE_TOPOLOGY_HWLOC) \
+       >> $(QTHREAD_INSTALL_DIR)/include/qthread-chapel.h
```

Comment:
Indentation looks a little off here. I also don't see a reason to put `CHPL_QTHREAD_TOPOLOGY_HWLOC` in if we're not using it.

Reply:
I found it useful for testing. I'll remove it.